src/wireviz/wv_helper.py

```diff
@@ -117,4 +117,12 @@ def open_file_read(filename):
     return open(filename, 'r', encoding='UTF-8')
 
 def open_file_write(filename):
-    return open(filename, 'w', encoding='UTF-8')
\ No newline at end of file
+    return open(filename, 'w', encoding='UTF-8')
+
+
+def manufacturer_info_field(manufacturer, mpn):
+    if manufacturer:
+        part_number_component = f': {mpn}' if mpn else ''
+        return(f'{manufacturer}{part_number_component}')
+    else:
+        return(f'MPN: {mpn}' if mpn else None)
```

Comment:
How about this?
```python
if manufacturer or mpn:
  return f'{manufacturer if manufacturer else "MPN"}{": " if mpn else ""}{mpn if mpn else ""}'
else
  return None
```

It should return the same four options:
`None` (nothing specified)
`{manufacturer}` (no MPN specified)
`MPN: {mpn}` (no manufacturer specified)
`{manufacturer}: {mpn}` (both specified)

Reply:
How about this?
```
    if manufacturer:
        return f'{manufacturer}: {mpn}' if mpn else manufacturer
    else:
        return f'MPN: {mpn}' if mpn else None
```

Reply:
That looks like a much tider way to implement that, have swapped to that implementation in the latest commits